backend/curriculum_tracking/api_views.py

```diff
@@ -1045,3 +1033,10 @@ class ContentItemAgileWeightViewSet(viewsets.ModelViewSet):
     serializer_class = serializers.ContentItemAgileWeightSerializer
     filter_backends = [DjangoFilterBackend]
     filterset_fields = ["content_item", "weight"]
+
+
+class CourseRegistrationViewset(viewsets.ModelViewSet):
+    queryset = models.CourseRegistration.objects.all().order_by("user")
+    serializer_class = serializers.CourseRegistrationSerialiser
+    filter_backends = [DjangoFilterBackend]
+    permission_classes = [core_permissions.IsReadOnly & permissions.IsAuthenticated]
```

Comment:
These permissions are a bit strange. Basically, any user can check what any other user is registered for so long as they are logged in.